### DISCUSSION TITLE
WIFI connection may have more advantages compared to Bluetooth connection. I don't know if the author has this idea

### DISCUSSION THREAD

**Discussion by peter123-cn** [TeamOpenFIRE/OpenFIRE-Firmware#39] [Ideas]
WIFI connection may have more advantages compared to Bluetooth connection. I don't know if the author has this idea

**Comment by SeongGino** [comment]
What Wifi connection? With what boards? With what chipsets? With what method?

Not even sure if I could say "feel free to submit a pull request" because I don't even know what this is trying to address...
